.github/workflows/tests.yml

```diff
@@ -154,10 +154,11 @@ jobs:
           pip install -U pip
           pip install -U setuptools wheel twine cffi
 
-      - name: Build BTrees (Python 3.10 on MacOS)
+      - name: Build BTrees (Python 3.10 and 3.11 on MacOS)
         if: >
           startsWith(runner.os, 'Mac')
-          && startsWith(matrix.python-version, '3.10')
+          && (startsWith(matrix.python-version, '3.10')
+              || startsWith(matrix.python-version, '3.11'))
         env:
           _PYTHON_HOST_PLATFORM: macosx-11-x86_64
```

Comment:
```suggestion
          _PYTHON_HOST_PLATFORM: macosx-10.15-x86_64
```

Reply:
Why 10.15?

Reply:
See test failure in GHA action "test (3.10, macos-latest)". There's actually more to do. The failure in "3.11 (macos-latest)" tells you there's more variables to be set so x86_64 gets built and used.

Reply:
Looks like the issue was in the condition you already fixed instead